### PR TITLE
Timur katimullin basic ellipsis https://basic-ellipsis.web.app/

### PR DESCRIPTION
**Pull request by linkouth**
Implements basic ellipsis mark up.

### CODE REVIEW

File: basic-ellipsis/public/index.html [hunk 1]
```diff
@@ -3,63 +3,113 @@
   <head>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1">
-    <title>Welcome to Firebase Hosting</title>
```
Comment (vovcyan) [inline basic-ellipsis/public/index.html:6]: Please add a title for your site

File: basic-ellipsis/public/index.html [hunk 2]
```diff
@@ -3,63 +3,113 @@
...
-        } catch (e) {
-          console.error(e);
-          document.getElementById('load').innerHTML = 'Error loading the Firebase SDK, check the console.';
-        }
-      });
-    </script>
+        <header class="wrapper">
+          <div class="container container__border_none">
+            <img src='images/Label.png' alt='label' class="header__logo" />
+            <h1 class="header__header">WORKING
```
Comment (vovcyan) [inline basic-ellipsis/public/index.html]: In 1 line

File: basic-ellipsis/public/index.html [hunk 3]
```diff
@@ -3,63 +3,113 @@
...
+              WITH design LAB </h1>
+            <p class="header__text">We’ve been helping startups launch and grow their businesses.
```
Comment (vovcyan) [inline basic-ellipsis/public/index.html]: Move text to next line

File: basic-ellipsis/public/index.html [hunk 4]
```diff
@@ -3,63 +3,113 @@
...
+              We help you to fill the gap between development, 
+                design and product management.</p>
+            <img src='images/Mouse-icon.svg' alt='scrollable' class="header__mouse-logo" />
```
Comment (vovcyan) [inline basic-ellipsis/public/index.html]: Please change alt

File: basic-ellipsis/public/index.html [hunk 5]
```diff
@@ -3,63 +3,113 @@
...
+          </div>
+        </header>
+        <div class="wrapper">
+          <div class="container container__color_black">
+            <h2 class="container__header">About us</h2>
+            <p class="container__text"> When you work with us, you are part of a team of committed collaborators.
```
Comment (vovcyan) [inline basic-ellipsis/public/index.html]: The same

File: basic-ellipsis/public/index.html [hunk 6]
```diff
@@ -3,63 +3,113 @@
...
+              product management to create exceptional products.
+              We start each project by understanding your business goals,
+              the user’s needs and the product requirements.
+              Then we help translate them into effective and elegant solutions.
+              </p>
+          </div>
+        </div>
+        <div class="wrapper wrapper__theme_services">
+          <div class="container container__services container__border_white">
+            <h2 class="container__header container__services__header container__header__background_none"><span>Services</span></h2>
```
Comment (vovcyan) [inline basic-ellipsis/public/index.html]: Please separate it into 2 lines

File: basic-ellipsis/public/index.html [hunk 7]
```diff
@@ -3,63 +3,113 @@
...
+            <p class="container__text">Before we sign a contract or write a line of code our team will spend the necessary
```
Comment (vovcyan) [inline basic-ellipsis/public/index.html]: The same

File: basic-ellipsis/public/index.html [hunk 8]
```diff
@@ -3,63 +3,113 @@
...
+              The outcome of this scoping session is a proposal or high-level statement of work.
+              Based on your requirements, we can provide a complete offering or just the pieces
+              you need. </p>
+          </div>
+        </div>
+        <div class="wrapper">
+          <div class="container container__color_black">
+            <h2 class="container__header">What we do</h2>
+            <div class="group">
+              <img src="images/DesignLogo.svg" class="group__label" alt="design logo" />
```
Comment (vovcyan) [inline basic-ellipsis/public/index.html]: The same

File: basic-ellipsis/public/index.html [hunk 9]
```diff
@@ -3,63 +3,113 @@
...
+              <h3 class="group__header">Design</h3>
+              <p class="group__text">Our design practice offers a full range of services including branding strategy,
```
Comment (vovcyan) [inline basic-ellipsis/public/index.html]: The same

File: basic-ellipsis/public/index.html [hunk 10]
```diff
@@ -3,63 +3,113 @@
...
+              <h3 class="group__header">Delevopment</h3>
+              <p class="group__text">All engineers are fluent in the latest enterprise, mobile and web development technologies.
+                They collaborate with your team to write, test, and improve code on a daily basis, using proven
+                practices such as test-driven development and pair programing.
+               </p>
+            </div>
+            <div class="group">
+              <img src="images/ProductManagmentLogo.svg" class="group__label" alt="product managment logo" />
+              <h3 class="group__header">Product managment</h3>
+              <p class="group__text">Planning and development is iterative. Because we are constantly coding and testing, the
```
Comment (vovcyan) [inline basic-ellipsis/public/index.html]: The same

File: basic-ellipsis/public/style/main.css [hunk 11]
```diff
@@ -0,0 +1,304 @@
...
+  padding: 0;
+  font-family: Lato;
+  font-size: 15px;
+  font-weight: 400;
+  line-height: 29px;
+  text-align: center;
+}
+
+.wrapper {
+  padding-top: 8vh;
```
Comment (vovcyan) [inline basic-ellipsis/public/style/main.css]: padding:

File: basic-ellipsis/public/style/main.css [hunk 12]
```diff
@@ -0,0 +1,304 @@
...
+  padding-left: 10vw;
+}
+
+.wrapper:first-child {
+  padding: 0;
+  padding-top: 26vh;
+}
+
+.wrapper:nth-last-child(2) {
+  padding-bottom: 0;
```
Comment (vovcyan) [inline basic-ellipsis/public/style/main.css]: The same

File: basic-ellipsis/public/style/main.css [hunk 13]
```diff
@@ -0,0 +1,304 @@
...
+.wrapper:last-child {
+  padding: 0;
+}
+
+.wrapper:nth-last-child(3) {
+  padding: 0;
+}
+
+.wrapper__theme_services {
+  background: url('../images/ServicesBG.png');
```
Comment (vovcyan) [inline basic-ellipsis/public/style/main.css]: -image

File: basic-ellipsis/public/style/main.css [hunk 14]
```diff
@@ -0,0 +1,304 @@
...
+  color: white;
+  white-space: pre-line;
+}
+
+.wrapper:first-child > .container {
+  padding: 0;
+}
+
+.wrapper:last-child > .container {
+  padding-top: 6vh;
```
Comment (vovcyan) [inline basic-ellipsis/public/style/main.css]: Why 6 vh?

File: basic-ellipsis/public/style/main.css [hunk 15]
```diff
@@ -0,0 +1,304 @@
...
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+  align-items: center;
+}
+
+header {
+  box-sizing: border-box;
+  height: 100vh;
+  background: url('../images/WelcomeBG.png');
```
Comment (vovcyan) [inline basic-ellipsis/public/style/main.css]: background-image:

File: basic-ellipsis/public/style/main.css [hunk 16]
```diff
@@ -0,0 +1,304 @@
...
+  line-height: normal;
+}
+
+.header__text {
+  margin-bottom: 20vh;
+  font-size: 15px;
+  font-weight: 400;
+  line-height: 29px;
+  text-transform: uppercase;
+  letter-spacing: 0.9px;
```
Comment (vovcyan) [inline basic-ellipsis/public/style/main.css:125]: .9px

File: basic-ellipsis/public/style/main.css [hunk 17]
```diff
@@ -0,0 +1,304 @@
...
+.group__text {
+  font-size: 14px;
+  font-weight: 300;
+  line-height: 25.89px;
+}
+
+.our-works {
+  margin-top: 20.5vh;
+  height: 47.7vh;
+  background-color: darkcyan;
```
Comment (vovcyan) [inline basic-ellipsis/public/style/main.css]: Please use HEX color

File: basic-ellipsis/public/style/main.css [hunk 18]
```diff
@@ -0,0 +1,304 @@
...
+}
+
+.footer__text {
+  margin-left: 2vw;
+  font-size: 14px;
+  font-weight: 400;
+}
+
+.wrapper:nth-child(5) {
+  padding-top: 0vh;
```
Comment (vovcyan) [inline basic-ellipsis/public/style/main.css]: padding:

File: basic-ellipsis/public/style/main.css [hunk 19]
```diff
@@ -0,0 +1,304 @@
...
+}
+
+.item:nth-child(2), .item:nth-child(5) {
+  flex-basis: 35vw;
+  margin-left: 10px;
+  margin-right: 10px;
+}
+
+.item:hover {
+  opacity: 0.5;
```
Comment (vovcyan) [inline basic-ellipsis/public/style/main.css]: .5;

File: basic-ellipsis/public/style/main.css [hunk 20]
```diff
@@ -0,0 +1,304 @@
...
+
+.item__text {
+  color: white;
+  font-size: 24px;
+  font-weight: 700;
+  text-transform: uppercase;
+  letter-spacing: 8.95px;
+}
+
+@media only screen and (max-width: 900px) {
```
Comment (vovcyan) [inline basic-ellipsis/public/style/main.css]: Please add media for vertical iPad

**Review by vovcyan — CHANGES_REQUESTED** [review]

File: basic-ellipsis/public/index.html [hunk 21]
```diff
@@ -1,65 +1,125 @@
...
-          let features = ['auth', 'database', 'messaging', 'storage'].filter(feature => typeof app[feature] === 'function');
-          document.getElementById('load').innerHTML = `Firebase SDK loaded with ${features.join(', ')}`;
-        } catch (e) {
-          console.error(e);
-          document.getElementById('load').innerHTML = 'Error loading the Firebase SDK, check the console.';
-        }
-      });
-    </script>
+        <header class="wrapper">
+          <div class="content_container content_container__without_border content_container__text_secondary_color">
```
Comment (vadeneev) [inline basic-ellipsis/public/index.html]: > __without_border
looks like modifier, but used as element  (according to syntax block-name__element_modifier)

File: basic-ellipsis/public/index.html [hunk 22]
```diff
@@ -1,65 +1,125 @@
...
+            <img src='images/Label.png' alt='label' class="header__logo" />
+            <h1 class="header__header">WORKING<br aria-hidden="true">WITH design LAB </h1>
```
Comment (vadeneev) [inline basic-ellipsis/public/index.html]: header__title ?

File: basic-ellipsis/public/index.html [hunk 23]
```diff
@@ -1,65 +1,125 @@
...
+               
+              Our approach unifies design, development and
+              product management to create exceptional products.
+              We start each project by understanding your business goals,
+              the user’s needs and the product requirements.
+              Then we help translate them into effective and elegant solutions.
+            </p>
+          </div>
+        </div>
+        <div class="wrapper wrapper__theme_services">
```
Comment (vadeneev) [inline basic-ellipsis/public/index.html]: looks like __theme_services is modifier

File: basic-ellipsis/public/index.html [hunk 24]
```diff
@@ -1,65 +1,125 @@
...
+          <div class="content_container content_container__services content_container__text_secondary_color content_container__border_secondary_color">
```
Comment (vadeneev) [inline basic-ellipsis/public/index.html]: `content_container__text_secondary_color content_container__border_secondary_color`
doesn't it look too... atomic ? could we use one color theme for example ?
(it is a question)

File: basic-ellipsis/public/index.html [hunk 25]
```diff
@@ -1,65 +1,125 @@
...
+            <h2 class="content_container__header content_container__services__header content_container__header__without_background"><span>Services</span></h2>
```
Comment (vadeneev) [inline basic-ellipsis/public/index.html]: there shouldn't be element of elements, means __element__element
__ = for element
_ = for modifier

File: basic-ellipsis/public/index.html [hunk 26]
```diff
@@ -1,65 +1,125 @@
...
+              The outcome of this scoping session is a proposal or high-level statement of work.
+              Based on your requirements, we can provide a complete offering or just the pieces
+              you need.
+            </p>
+          </div>
+        </div>
+        <div class="wrapper">
+          <div class="content_container">
+            <h2 class="content_container__header">What we do</h2>
+            <div class="group">
```
Comment (vadeneev) [inline basic-ellipsis/public/index.html]: Doesn't this one looks as like as content-container ?
(same stuff but without border and with image)

File: basic-ellipsis/public/index.html [hunk 27]
```diff
@@ -1,65 +1,125 @@
...
+            <p class="content_container__text">
+              Planning and development is iterative. Because we are constantly coding and
+              testing, the products we build are always ready to go live. This iterative process
+              allows for changes as business requirements evolve.
+            </p>
+            <button class="contact-us__button">Contact us</button>
+          </div>
+        </div>
+        <footer class="wrapper">
+          <div class="content_container content_container__without_border footer__content_container">
```
Comment (vadeneev) [inline basic-ellipsis/public/index.html]: can't see where footer block starts

File: basic-ellipsis/public/style/main.css [hunk 28]
```diff
@@ -0,0 +1,324 @@
...
+}
+
+.content_container__services__header:before, .content_container__services__header:after {
+  white-space: nowrap;
+  content: "";
+  border-top: 2px solid white;
+  width: 50%;
+}
+
+.content_container__services__header > span {
```
Comment (vadeneev) [inline basic-ellipsis/public/style/main.css]: we should not use elements in BEM, also there is no place for element of element

File: basic-ellipsis/public/style/main.css [hunk 29]
```diff
@@ -0,0 +1,324 @@
...
+
+.wrapper:first-child > .content_container {
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+  align-items: center;
+}
+
+header {
```
Comment (vadeneev) [inline basic-ellipsis/public/style/main.css]: we should not use element selector in BEM

**Review by vadeneev — CHANGES_REQUESTED** [review]

**Review by vovcyan — DISMISSED** [review]

**Review by vovcyan — APPROVED** [review]

**Review by vadeneev — APPROVED** [review]

**Pull request merged**